app/views/layouts/application.html.erb

```diff
@@ -38,6 +38,8 @@
       </nav>
     </header>
 
+    <%= render 'shared/flashes' %>
+
```

Comment:
this is the pattern I saw in a bunch of access team apps.  it seemed like a great way to go.